kale/predict/decode.py

```diff
@@ -32,3 +43,122 @@ def forward(self, x):
         x = F.relu(self.fc3(x))
         x = self.fc4(x)
         return x
+
+
+class MultiRelaInnerProductDecoder(torch.nn.Module):
```

Comment:
Have you considered call it DistMult? Unless different. Think about what if others give a longname to gripnet and do not use the name gripnet.

Reply:
Also I assume it will be covered in the test by GripNetLinkPrediction( ), right?

Reply:
Yes, it is covered by the test of `GripNetLinkPrediction()`.

Reply:
I renamed it to `DistMultDecoder` to distinguish it from the original DistMult model, which contains an additional encoder.

Reply:
Can you add docstrings for its arguments `Args:`? See `MLPDecoder` for an example.